service/routes.py

```diff
@@ -17,17 +19,66 @@
 ######################################################################
 @app.route("/")
 def index():
-    """ Root URL response """
+    """ Promotion's Home URL response """
+    app.logger.info("Request for Promotion's Home Root URL")
     return (
-        "Reminder: return some useful information in json format about the service here",
-        status.HTTP_200_OK,
+        jsonify(
+            name="Promotion Rest API Service!!",
+            version="1.0"
+        ),
+        status.HTTP_200_OK
     )
-
+    
 
 ######################################################################
 #  R E S T   A P I   E N D P O I N T S
 ######################################################################
 
+# Place your REST API code here ...
+
+
+
+
+######################################################################
+# ADD A NEW Promotion
+######################################################################
+@app.route("/promotions", methods=["POST"])
+def create_promotion():
+    """
+    Creates a Promotion
+    This endpoint will create a Promotion based the data in the body that is posted
+    """
+    
+    app.logger.info("Request to create a promotion")
+    check_content_type("application/json")
+    
+    promo = Promotion()
+    promo.deserialize(request.get_json())
+    promo.create()
+    message = promo.serialize()
+    
+
+    app.logger.info("Promotion with ID [%s] created.", promo.id)
+    return jsonify(message), status.HTTP_201_CREATED
+
+
+def check_content_type(content_type):
```

Comment:
add a comment stating utility function

Reply:
A helper function should be self-explanatory. Comments are only necessary for things that are not straightforward, for example, you use a special regex or doing something unusual due to business logic.

The best practice for helper function is to put it in a separate file utils.py or put it at the bottom of the file.